### DISCUSSION TITLE
Time for a new release?

### DISCUSSION THREAD

**Discussion by tessus** [orhun/rustypaste#313] [General]
Hi @orhun. I think it's time for a new release. There weren't too many changes, apart from lot of dependency updates, but it would be nice to have #294 available in a release.

Edit: The cli could also need a new release.

What do you think?

**Comment by orhun** [comment]
Hey @tessus! Just got back from my time-off. Yes, I think it's a time for a release.

Will get to that in the following days, thanks for the reminder!

**Reply by tessus** [reply]
Great, maybe you can also have a look at my comments in #315

**Reply by tessus** [reply]
Can you also create a new releases for the cli?

P.S.: If there is anything I can do to lessen the load, please let me know.

**Reply by orhun** [reply]
done!

**Reply by tessus** [reply]
Awesome, thanks!
